cdap-hbase-compat-1.1/src/test/java/co/cask/cdap/data2/increment/hbase11/IncrementHandlerTest.java

```diff
@@ -0,0 +1,206 @@
+      assertEquals(expected[i], Bytes.toLong(scanResCell.getValue()));
+    }
+  }
+
+  @Override
+  public HTable createTable(TableId tableId) throws Exception {
+    HBaseTableUtil tableUtil = new HBaseTableUtilFactory(cConf).get();
+    HTableDescriptorBuilder tableDesc = tableUtil.buildHTableDescriptor(tableId);
+    HColumnDescriptor columnDesc = new HColumnDescriptor(FAMILY);
+    columnDesc.setMaxVersions(Integer.MAX_VALUE);
+    columnDesc.setValue(IncrementHandlerState.PROPERTY_TRANSACTIONAL, "false");
+    tableDesc.addFamily(columnDesc);
+    tableDesc.addCoprocessor(IncrementHandler.class.getName());
+    HTableDescriptor htd = tableDesc.build();
+    testUtil.getHBaseAdmin().createTable(htd);
+    testUtil.waitUntilTableAvailable(htd.getName(), 5000);
+    return tableUtil.createHTable(conf, tableId);
+  }
+
+  @Override
+  public RegionWrapper createRegion(TableId tableId, Map<String, String> familyProperties) throws Exception {
+    HColumnDescriptor columnDesc = new HColumnDescriptor(FAMILY);
+    columnDesc.setMaxVersions(Integer.MAX_VALUE);
+    for (Map.Entry<String, String> prop : familyProperties.entrySet()) {
+      columnDesc.setValue(prop.getKey(), prop.getValue());
+    }
+    return new HBase98RegionWrapper(
+        IncrementSummingScannerTest.createRegion(testUtil.getConfiguration(), cConf, tableId, columnDesc));
+  }
+
+  public static ColumnCell convertCell(Cell cell) {
+    return new ColumnCell(CellUtil.cloneRow(cell), CellUtil.cloneFamily(cell), CellUtil.cloneQualifier(cell),
+        cell.getTimestamp(), CellUtil.cloneValue(cell));
+  }
+
+  public class HBase98RegionWrapper implements RegionWrapper {
+    private final HRegion region;
+
+    public HBase98RegionWrapper(HRegion region) {
+      this.region = region;
+    }
+
+    @Override
+    public void initialize() throws IOException {
+      region.initialize();
+    }
+
+    @Override
+    public void put(Put put) throws IOException {
+      region.put(put);
+    }
+
+    @Override
+    public boolean scanRegion(List<ColumnCell> results, byte[] startRow) throws IOException {
+      return scanRegion(results, startRow, null);
+    }
+
+    @Override
+    public boolean scanRegion(List<ColumnCell> results, byte[] startRow, byte[][] columns) throws IOException {
+      Scan scan = new Scan().setMaxVersions().setStartRow(startRow);
+      if (columns != null) {
+        for (int i = 0; i < columns.length; i++) {
+          scan.addColumn(FAMILY, columns[i]);
+        }
+      }
+      RegionScanner rs = region.getScanner(scan);
+      try {
+        List<Cell> tmpResults = new ArrayList<>();
+        boolean hasMore = rs.next(tmpResults);
+        for (Cell cell : tmpResults) {
+          results.add(convertCell(cell));
+        }
+        return hasMore;
+      } finally {
+        rs.close();
+      }
+    }
+
+    @Override
+    public boolean flush() throws IOException {
+      HRegion.FlushResult result = region.flushcache(true, false);
```

Comment:
It would be a good idea to add a comment here saying why `flushCache` is `(true, false)`.